src/Paket.Core/ProjectFile.fs

```diff
@@ -129,8 +129,6 @@ type ProjectFile =
             node.ParentNode.RemoveChild(node) |> ignore
 
     member this.UpdateFileItems(fileItems : FileItem list, hard) = 
-        this.DeletePaketNodes("Compile")
```

Comment:
the idea was: remove all paket stuff from a fsproj file and add it again. since we can't know if someone remove a package.